src/gui/accountstate.cpp

```diff
@@ -422,6 +450,10 @@ void AccountState::fetchNavigationApps(){
     job->getNavigationApps();
 }
 
+void AccountState::fetchUserStatus() {
+    _userStatus->fetchUserStatus(AccountStatePtr(this));
```

Comment:
Couldn't that get a raw pointer or a pointer to Account instead? Creating a shared pointer from `this` is generally asking for troubles.